### ISSUE TITLE
Form changes should reflect on the canvas

### ISSUE THREAD

**Issue by lordrip** [KaotoIO/kaoto#720]
### Please describe the task that needs to be done

Since the description field is used as a label for the canvas nodes, it would be ideal to have them updated as soon as the form edit happens.

**Comment by lhein** [comment]
I see a merged PR in the history. Is that fixing this issue, @lordrip ?

**Comment by lordrip** [comment]
> I see a merged PR in the history. Is that fixing this issue, @lordrip ?

No, it's not, the PR link comes [from my message](https://github.com/KaotoIO/kaoto-next/pull/711#issuecomment-1908274821) mentioning two topics we need to take into account for the next step
